Classes/Common/MetsDocument.php

```diff
@@ -333,7 +333,7 @@ protected function getLogicalStructureInfo(\SimpleXMLElement $structure, $recurs
         $details['orderlabel'] = (isset($attributes['ORDERLABEL']) ? $attributes['ORDERLABEL'] : '');
         $details['contentIds'] = (isset($attributes['CONTENTIDS']) ? $attributes['CONTENTIDS'] : '');
         $details['volume'] = '';
-        // Set volume information only if no label is set and this is the toplevel structure element.
+        // Set volume any year information only if no label is set and this is the toplevel structure element.
         if (
             empty($details['label'])
             && $details['id'] == $this->_getToplevelId()
```

Comment:
Should we really limit this to toplevel elements? It could be confusing if I'd configured some replacements in the ToC plugin but they don't work because the respective elements are not toplevel...

I suggest dropping this condition and just checking for empty labels and orderlabels:
```suggestion
            && empty($details['orderlabel'])
```